Danki2/Assets/Scripts/Abilities/InstantCast/InstantCast.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Abilities;
+
+namespace Assets.Scripts.Abilities.InstantCast
```

Comment:
I'm not sure I really understand namespaces, I was thinking this would just be `namespace Abilities`? I think if you have it like this you'd have to do something like `using InstantCast.InstantCast` which doesn't seem right?

Reply:
I agree with you in principle, in practice I always just leave them as defaults which means they are simply aligned with the file tree.

Reply:
Sounds good to me